worker/rhpv2.go

```diff
@@ -372,7 +372,9 @@ func (w *worker) DeleteContractRoots(ctx context.Context, hostIP string, hostKey
 			})
 
 			// check funds
-			cost := rhpv2.RPCDeleteCost(settings, len(batch))
+			proofSize := 32 * (128 + uint64(len(batch)))
+			cost := settings.BaseRPCPrice.Add(settings.DownloadBandwidthPrice.Mul64(proofSize))
+			cost = cost.Mul64(105).Div64(100)
```

Comment:
Where does this leave us with regards to updating `core` to have exact cost calculations in `renterd` and `hostd`?

Reply:
I opened <https://github.com/SiaFoundation/core/pull/123> but I think we can merge this and update this later